main/management/commands/check_refresh_tokens.py

```diff
@@ -0,0 +1,11 @@
+from django.core.management.base import BaseCommand
+from main.models import FitbitMember
+
+
+class Command(BaseCommand):
+    help = 'Update data for all users'
```

Comment:
seems like this doesn't match the name of the management command itself (`check_refresh_token`)? :)